### PR TITLE
EN-1742-1695-MASTER-save-new-profile-type

### PR DESCRIPTION
**Pull request by DMinnai**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/state/profile-types/actions.js [hunk 1]
```diff
@@ -162,9 +161,12 @@ export const sendPostProfileType = ProfileTypeObject => dispatch =>
     postProfileType(ProfileTypeObject).then((response) => {
       response.json().then((json) => {
         if (response.ok) {
+          dispatch(addToast(formattedText('ProfileType.created'), TOAST_SUCCESS));
           gotoRoute(ROUTE_PROFILE_TYPE_EDIT, { profiletypeCode: json.payload.code });
         } else {
           dispatch(addErrors(json.errors.map(err => err.message)));
+          json.errors.forEach(err => dispatch(addToast(err.message, TOAST_ERROR)));
+          dispatch(clearErrors());
```
Comment (fedelocci) [inline src/state/profile-types/actions.js]: Why this ?

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**